guides/common/modules/con_limitations-of-installation-in-an-ipv6-network.adoc

```diff
@@ -0,0 +1,7 @@
+[id="limitations-of-installation-in-an-ipv6-network_{context}"]
+= Limitations of {Project} Installation in an IPv6 Network 
+
+{Project} installation in an IPv6 network has the following limitations:
+
+* DHCP server in {RHEL} (ISC DHCP) does not provide an integration API for managing IPv6 records, therefore {SmartProxy} DHCP plug-in that provides DHCP management is limited to IPv4 subnets. In this scenario, DHCP IPv6 must be deployed as a separate unmanaged service to bootstrap client nodes into grub2, which then configures IPv6 networking either using DHCPv6 or via static IPv6 address.
+* Although provisioning templates which ship with {Project} include IPv6 support for PXE and HTTP (iPXE) scenarios, the only tested and certified workflow is UEFI HTTP Boot.
```

Comment:
This is limitation for provisioning, not for generic use on IPv6. You can still install and use project on IPv6 without provisioning just fine - e.g. syncing content, calling remote execution jobs etc.